apps/web/src/features/myAccounts/hooks/useGetHref.ts

```diff
@@ -23,7 +23,7 @@ export const useGetHref = (router: NextRouter) => {
             : isSingleTxPage
               ? AppRoutes.transactions.history
               : router.pathname,
-        query: { ...router.query, safe: `${chain.shortName}:${address}` },
+        query: { ...(!isSpacePage && router.query), safe: `${chain.shortName}:${address}` },
```

Comment:
Should this just pass a `safe` param? What other params does it need?

Reply:
The `...router.query` was there before so I didn't want to touch it but at the same time I don't want to keep the `spaceId` param that we show when on a space and if the user then clicks on one of the safe items so I had to add this exception.